src/Raven.Server/Documents/TransactionOperationsMerger.cs

```diff
@@ -268,6 +272,7 @@ public override void Prepare(ref RecordingState state)
                         return;
                     }
 
+                    //Todo To decide where the file should be saved
```

Comment:
What do you mean? You got the path from the user or this, no?

Reply:
Yes, in a local machine it's work perfect.
But if I'll do it in remote? Should the client send the whole path?

Reply:
It is always a local path, yes. And if sent from remote machine, the caller need to provide the right path.